.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  pull_request:
```

Comment:
AFAIK

```
  pull_request:
    types: [opened, synchronize]
```

should suffice, i.e. we don’t need to re-run the build for other PR actions

Reply:
I can change this.

Out of curiosity: Are there any other actions? (Closing a PR does not trigger a build AFAIK.)

Reply:
There are the following `pull_request` activity types:

- assigned
- unassigned
- labeled
- unlabeled
- opened
- edited
- closed
- reopened
- synchronize
- ready_for_review
- locked
- unlocked
- review_requested
- review_request_removed

But apparently when you don’t specify `types` explicitly, only `opened`, `synchronize` and `reopened` trigger builds so we’d only get rid of `reopened` so I guess you can leave it as-is.

(Though I would assume reopened PRs still have their previous builds lying around so I don’t know if there’s a need to actually rebuild but since reopens are relatively rare, it shouldn’t matter).